Source/WebKit/UIProcess/Extensions/Cocoa/API/WebExtensionContextAPIActionCocoa.mm

```diff
@@ -96,7 +96,7 @@
         return;
     }
 
-    completionHandler(action.value()->label(WebExtensionAction::FallbackWhenEmpty::No));
+    completionHandler(Ref { action.value() }->label(WebExtensionAction::FallbackWhenEmpty::No));
```

Comment:
Do we really need referencing action.value()? `action` is `Expected<Ref<WebExtensionAction>, WebExtensionError>` and already in stack and `ref()`ed.

Reply:
Offline discussion with Ryosuke, we've decided to protect the Ref value explicitly on stack to settle static analyzer warning for now. In the future, we'll introduce exception for Expect and std::optional.